### ISSUE TITLE
Error on running example with yarn

### ISSUE THREAD

**Issue by keul** [transitive-bullshit/create-react-library#23]
Just created the app, then followed CLI instruction.

In one shell entered the new project the run `yarn start` (ok)

In the other shell entered `/example` and run `yarn start` but I get this:

```
$ yarn start
yarn run v1.6.0
$ react-scripts start
/bin/sh: react-scripts: command not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by keul** [comment]
...obviously fixed by manually running `yarn install` inside example folder.

**Comment by transitive-bullshit** [comment]
Thanks for the head's up @keul.

This was due to a recent change; just pushed a fix in `v2.3.8`.  Thanks again!
